src/linkgraph/refresh.cpp

```diff
@@ -203,6 +203,7 @@ void LinkRefresher::RefreshStats(const Order *cur, const Order *next)
 	StationID next_station = next->GetDestination();
 	Station *st = Station::GetIfValid(cur->GetDestination());
 	if (st != nullptr && next_station != INVALID_STATION && next_station != st->index) {
+		Station *st_to = Station::Get(next_station);
```

Comment:
Can `st_to` ever be a nullptr?
Line 215 accesses it directly, whie line 225 checks the pointer first.
Possibly also remove the check in line 225?